Tests/LibWeb/Text/input/checkbox-focus-lost-no-change-event.html

```diff
@@ -2,15 +2,22 @@
 <input type="checkbox" id="checkbox">
 <script src="include.js"></script>
 <script>
+    let changeEventFired = false;
     checkbox.addEventListener("change", () => {
-      println("Change event was fired when it shouldn't have been.");
+      changeEventFired = true;
     });
     asyncTest(async done => {
         checkbox.focus();
         await new Promise(resolve => setTimeout(resolve, 0));
         checkbox.blur();
         await new Promise(resolve => setTimeout(resolve, 0));
 
+        if (changeEventFired) {
+            println("FAIL: Change event was fired when it shouldn't have been.");
+        } else {
+            println("PASS: Change event was not fired");
+        }
+
         done();
     });
-</script>
\ No newline at end of file
+</script>
```

Comment:
Can you put these changes in a separate commit, please.

Reply:
Yes, done, sorry.